src/main/kotlin/org/jitsi/nlj/transform/node/incoming/AudioLevelReader.kt

```diff
@@ -52,10 +55,13 @@ class AudioLevelReader(
         audioLevelExtId?.let { audioLevelId ->
             audioRtpPacket.getHeaderExtension(audioLevelId)?.let { ext ->
                 val level = AudioLevelHeaderExtension.getAudioLevel(ext)
-                if (level == MUTED_LEVEL || this.forceMute) {
+                if ((level == MUTED_LEVEL && forwardedSilencePackets > forwardedSilencePacketsLimit) ||
+                    this.forceMute
+                ) {
                     packetInfo.shouldDiscard = true
                     numSilencePacketsDiscarded++
                 } else {
+                    forwardedSilencePackets = if (level == MUTED_LEVEL) +1 else 0
```

Comment:
I think `+1` should be `forwardedSilencePackets + 1`

Reply:
You are definitely right! Stupid mistake..